deployments/sepolia_1701265089/deploy_output.json

```diff
@@ -0,0 +1,26 @@
+{
+ "polygonZkEVMAddress": "0xB6dA7da9FB1dd26e48Dde908A3391506FB8FC479",
+ "polygonZkEVMBridgeAddress": "0x78E53eC2d9E71E4af3c942D04C69e3F37f81DD15",
+ "polygonZkEVMGlobalExitRootAddress": "0xb91F4Ef2afB5fB7b3de4822E69E00c4638C99107",
+ "forkingManager": "0xA57449116f66d77Dc4ad7221B9834A91FdE2E933",
```

Comment:
Don't necessarily need to fix it in this commit but I noticed
that we have `forkingManager` on its own but everything else is `somethingAddress`

Reply:
true... we can fix this in the deployment script, but not in this PR.